.github/workflows/python-package.yml

```
@@ -30,7 +30,7 @@ jobs:
         python -m pip install pylint pytest pandas
```

To install the package locally:

```suggestion
        python -m pip install pylint pytest
        pip install .
```

This also takes care of installing the package dependencies

---

Yes, it worked. Thank you!